test/smtcallback.js

```diff
@@ -146,6 +147,11 @@ tape('SMTCheckerCallback', function (t) {
         continue;
       }
 
+      if (source.includes('// SMTIgnoreCex: yes')) {
+        st.skip('Test ignores counterexamples.');
+        continue;
+      }
```

Comment:
So you don't bother to just ignore the counterexamples here, but skip the entire test :-)?
You *could* just cut the expectations short after "happens here" in those cases, couldn't you?

Reply:
Yes, I found that easier. It's only a handful of tests and the counterexamples are tested on the rest so, no harm I'd say. Just trying to keep it simple. On the other hand, I could also do that, yes.

Reply:
Updated